zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -131,8 +131,12 @@ public void onMessage(NotebookSocket conn, String msg) {
       }
       
       String ticket = TicketContainer.instance.getTicket(messagereceived.principal);
-      if (ticket != null && !ticket.equals(messagereceived.ticket))
-        throw new Exception("Invalid ticket " + messagereceived.ticket + " != " + ticket);
+      if (ticket != null && !ticket.equals(messagereceived.ticket)){
+        /* not to pollute logs, log in debug instead of exception */
+        LOG.debug("{} message: invalid ticket {} != {}", messagereceived.op,
```

Comment:
When messagereceived.ticket is null or empty string, which is the case user not logged in and browse Zeppelin GUI, i think we can minimize the log in this case (log.debug?).

When messagereceived.ticket is not empty, but not equals to ticket, that means something wrong, and in this it's better print some logs.